Scripts/developer_scripts/cgal_create_release_with_cmake.cmake

```diff
@@ -61,8 +64,17 @@ foreach(pkg ${files})
         if(NOT IS_DIRECTORY ${release_dir}/${afile_dir})
           file(MAKE_DIRECTORY ${release_dir}/${afile_dir})
         endif()
-        #copy the file
-        file(COPY ${afile} DESTINATION ${release_dir}/${afile_dir})
+
+        #copy the file (replace $URL$ and $ID$ for *.h and *.hpp)
+        get_filename_component(fext ${fname} EXT)
+        if ("${fext}" STREQUAL ".h" OR "${fext}" STREQUAL ".hpp")
+          file(READ "${pkg_dir}/${f}" file_content)
+          string(REPLACE "$URL$" "${GITHUB_PREFIX}/${pkg}/${f}" file_content ${file_content})
+          string(REPLACE "$Id$" "This file is from the release ${CGAL_VERSION} of CGAL" file_content ${file_content})
+          file(WRITE ${release_dir}/${afile_dir}/${fname} ${file_content})
+        else()
+          file(COPY ${afile} DESTINATION ${release_dir}/${afile_dir})
+        endif()
```

Comment:
1. Does not this slow down the script a lot?
2. The `$URL` and `$Id$` must be replaced by something starts with `$URL: ` or `$Id: ` and ends with `$`.

Reply:
The script is pretty fast (still way faster than the old one even on my laptop).
Do you mean I should replace by `$URL: https://www.github.com/....`?

Reply:
By `$URL: https://www.github.com/....$`

Reply:
`$Id$` is supposed to be turned into `$Id: <id>$` when `<id>` is a "summary of the other information":
http://svnbook.red-bean.com/en/1.7/svn.advanced.props.special.keywords.html

Probably to be used by tool to designate the file. In the context of Git, it could be the full filename, plus the result of `git describe HEAD`.